### ISSUE TITLE
Server require error.

### ISSUE THREAD

**Issue by rdtomar** [jquense/yup#270]
I am trying to add yup on my server but getting this error, please help!!!
```
internal/modules/cjs/loader.js:596
    throw err;
    ^

Error: Cannot find module '@babel/runtime/helpers/builtin/interopRequireDefault'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:594:15)
    at Function.Module._load (internal/modules/cjs/loader.js:520:25)
    at Module.require (internal/modules/cjs/loader.js:650:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/aadii104/companies/proworktree/proworktree-server/node_modules/yup/lib/index.js:3:30)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:713:10)
```

**Comment by kjendrzyca** [comment]
tldr: downgrade `@babel/runtime` version -> `yarn add @babel/runtime@7.0.0-beta.55`

`@babel/runtime` was updated to beta56 where they changed the folder structure. One of the packages in the tree uses `@babel/runtime` and for some reason this beta version is not frozen, it is set up to update automatically. Strange call when using beta of a lib that can break anytime.

**Comment by jquense** [comment]
fixed in `0.26.1`
